.github/workflows/build.yml

```
@@ -11,10 +11,10 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - name: Set up JDK 1.8
+      - name: Set up JDK 17
         uses: actions/setup-java@v1
```

actionlint (1.7.7)

the runner of "actions/setup-java@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)